dev/ansible/05_presto_create_tables.yml

```
@@ -0,0 +1,50 @@
+- name: Presto healthchecks pass
+  uri:
+    url: http://localhost:8500/v1/health/checks/presto?filter={{ '(Status=="passing")' | urlencode }}
+    method: GET
+    return_content: yes
+    status_code: 200
+    body_format: json
+  register: result_presto
+  retries: 15
+  delay: 15
+  until: result_presto.json | length == 3
+  tags: example-upload
```

```suggestion
- name: Presto healthchecks pass
  uri:
    url: http://localhost:8500/v1/health/checks/presto
    method: GET
    return_content: yes
    status_code: 200
    body_format: json
  register: result_presto
  retries: 15
  delay: 15
  until: '"critical" not in result_presto | string'
  tags: example-upload
```

Could we do something like this? More dynamic

---

I would like to stay explicit and not dynamic. 3 healthchecks should have status - passing

---

I don't have too much of a preference, so that's completely fine with me.